run.py

```diff
@@ -237,6 +237,7 @@ def create_nodes(
                     private_ip=private_ip,
                     hostname=node.hostname,
                     ceph_vmnode=node,
+                    ceph_nodename=node.node.name,
```

Comment:
From where this `node.node.name` coming from ? I see `node-name` variable here - https://github.com/red-hat-storage/cephci/blob/ebfe67ee0fab06bdd242662df4d5466f22e410dc/ceph/utils.py#L340

Reply:
Not sure, how it is! But from the dir it was coming like that. Here are the logs: http://magna002.ceph.redhat.com/cephci-jenkins/aramteke/automation-83575600/cephci-run-RQD2U9/